### PR TITLE
re-designed header and nav bar

### PR DESCRIPTION
**Pull request by ghosharnab969**
previous design : 
image

new design : 
image

### CODE REVIEW

File: css/pages/index.css [hunk 1]
```diff
@@ -2,7 +2,180 @@
...
+div.heading-container {
+  flex-direction: column;
+  display: inline-block;
+  position: relative;
+  top: 5em;
+  left: 2em;
+}
+#first-line {
+  font-size: 4.3em;
+  line-height: -3em;
```
Comment (harshith-venkatesh) [inline css/pages/index.css:78]: What is thr purpose of negetive libe height

File: css/pages/index.css [hunk 2]
```diff
@@ -2,7 +2,180 @@
...
+  width: 9em;
+  height: 1em;
+  display: inline-block;
+  position: relative;
+  top: 0.8em;
+  left: 0.8em;
+}
+#welcome-text {
+  font-family: 'raleway';
```
Comment (harshith-venkatesh) [inline css/pages/index.css:87]: Is there a way we can wrap a div with a font family instead of repetition

File: index.html [hunk 3]
```diff
@@ -14,14 +14,190 @@
...
+          >
+          <a
+            class="nav-text crypto"
+            href="https://www.google.com"
+            target="_blank"
+            >Crypto</a
+          >
+          <a
+            class="nav-text status"
+            href="https://www.google.com"
```
Comment (harshith-venkatesh) [inline index.html:81]: Href is not addressed properly here

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: css/pages/index.css (hunk 1 above)
Comment (ghosharnab969) [reply css/pages/index.css:78]: I have removed the line-height attribute, thank you for your review!

File: css/pages/index.css (hunk 2 above)
Comment (ghosharnab969) [reply css/pages/index.css:87]: yes, there is definitely, and I corrected it as well!

File: index.html (hunk 3 above)
Comment (ghosharnab969) [reply index.html:81]: The re-designing of the whole website is yet to complete, and tasks have been assigned to other members who will be working on this code after this PR gets accepted. So hopefully, the links will be addressed properly once the whole re-designing process is complete.

File: css/pages/index.css [hunk 4]
```diff
@@ -2,7 +2,177 @@
   box-sizing: border-box;
 }
 
-.container {
+/* Arnab's code */
+body {
+  margin: 0;
+}
```
Comment (Rucha1499) [inline css/pages/index.css:8]: Kindly leave one line after writing CSS for a class/id. Code will not look crowded that way! 😃

Comment (harshith-venkatesh) [reply css/pages/index.css:8]: We don't comment that it is our code
GitHub lens automatically mentions in whose comments these changes are committed

File: index.html [hunk 5]
```diff
@@ -14,14 +14,190 @@
...
+        <img
+          id="arrow-icon"
+          src="../img\re-designed images\arrow icon.png"
+          alt="arrow icon"
+        />
+      </div>
+    </div>
+    <div class="second-screen"></div>
+    <div class="third-screen"></div>
+    <!-- <header> -->
```
Comment (harshith-venkatesh) [inline index.html:197]: Comments can be removed if we are not using it

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: css/pages/index.css (hunk 4 above)
Comment (Pujarini) [reply css/pages/index.css:8]: Harshith actually we kept the old css for now we will remove it while integrating all the components of the screen

File: css/pages/index.css [hunk 6]
```diff
@@ -2,7 +2,177 @@
...
+  font-family: 'poppins';
+  font-size: 1em;
+  box-sizing: border-box;
+}
+.new {
+  width: 5rem;
+  height: 2.8rem;
+  background: linear-gradient(130deg, #f3036b, #dc7faf, #e30062);
+  color: #091133;
+  font-family: 'poppins';
```
Comment (Pujarini) [inline css/pages/index.css:160]: Please use shorthand for font
https://developer.mozilla.org/en-US/docs/Web/CSS/font

File: css/pages/index.css [hunk 7]
```diff
@@ -60,3 +60,179 @@
...
+  top: 0.8em;
+  left: 0.8em;
+}
+#welcomeTxt {
+  font-family: 'raleway';
+  color: #e30062;
+}
+#to-theTxt {
+  font-family: 'raleway';
+  color: #fff;
```
Comment (Pujarini) [inline css/pages/index.css:90]: use shorthand for font here too.

File: css/pages/index.css [hunk 8]
```diff
@@ -60,3 +60,179 @@
...
+  font-weight: 400;
+}
+#realDevSquadTxt {
+  color: #fff;
+  font-family: 'raleway';
```
Comment (Pujarini) [inline css/pages/index.css]: shorthand for font here too

File: css/pages/index.css [hunk 9]
```diff
@@ -60,3 +60,179 @@
...
+  font-weight: 700;
+  font-size: 2.3em;
+  line-height: 0.6em;
+  position: relative;
+  left: 0.8em;
+  top: 1.2em;
+}
+div.heading-container {
+  flex-direction: column;
+  display: inline-block;
```
Comment (Pujarini) [inline css/pages/index.css]: display is inline-block but you are using flex-direction?  Why so?

File: index.html [hunk 10]
```diff
@@ -14,14 +14,190 @@
...
+            class="nav-text events"
+            href="https://www.google.com"
+            target="_blank"
+            >Events</a
+          >
+          <a
+            class="nav-text member"
+            href="https://www.google.com"
+            target="_blank"
+            >Member</a
```
Comment (Pujarini) [inline index.html:71]: Add the actual routes

**Review by Pujarini — CHANGES_REQUESTED** [review]

File: css/pages/index.css (hunk 4 above)
Comment (harshith-venkatesh) [reply css/pages/index.css:8]: I am mentioning line 5 here

Comment (ghosharnab969) [reply css/pages/index.css:8]: > Kindly leave one line after writing CSS for a class/id. Code will not look crowded that way! 😃

I have corrected the code.

File: css/pages/index.css (hunk 6 above)
Comment (ghosharnab969) [reply css/pages/index.css:160]: > Please use shorthand for font
> https://developer.mozilla.org/en-US/docs/Web/CSS/font

I have corrected the code and used short-hand wherever possible. In some places using font short-hand is leading to repetition of code, so I have left those sections unchanged!

File: css/pages/index.css (hunk 9 above)
Comment (ghosharnab969) [reply css/pages/index.css]: > display is inline-block but you are using flex-direction? Why so?

It seems I forgot to delete the flex-direction property while manipulating the code. I have fixed the issue now!

File: index.html (hunk 10 above)
Comment (ghosharnab969) [reply index.html:71]: > Add the actual routes
actual routes added!

File: index.html [hunk 11]
```diff
@@ -14,119 +14,157 @@
...
 
+      <img
+        class="welcome-image"
+        src="../img\re-designed images\welcome-image.png"
+        alt="welcome-image"
+      />
+      <div class="news-feeds-container">
+        <div class="new-tag">New</div>
+        1 Year Aniversary--New Features Available for YOU!
+        <a id="checkout-tag" href="https://www.google.com">Check them out </a>
```
Comment (Rucha1499) [inline index.html:160]: Can we have this link replaced by this? `https://www.realdevsquad.com/releases`

**Review by Rucha1499 — CHANGES_REQUESTED** [review]

File: index.html (hunk 11 above)
Comment (ghosharnab969) [reply index.html:160]: > Can we have this link replaced by this? `https://www.realdevsquad.com/releases`

Sure!

Comment (ghosharnab969) [reply index.html:160]: > Can we have this link replaced by this? `https://www.realdevsquad.com/releases`

sure!

**Review by Rucha1499 — APPROVED** [review]
Great work! @ghosharnab969 🎉

**Pull request merged**